### ISSUE TITLE
mobilenet.h5转换KeyError: 'threshold'错误

### ISSUE THREAD

**Issue by ncnnnnn** [MarsTechHAN/keras2ncnn#15]
H5文件：
```
https://github.com/lutzroeder/netron/ 的 Keras: mobilenet 
```

命令：
```
python3 -m keras2ncnn -i ~/Downloads/mobilenet.h5  -o ./
```

出错信息
```python
 File "/home/hx/.local/lib/python3.6/site-packages/keras2ncnn/keras2ncnn.py", line 67, in main
    KerasConverter().parse_keras_graph(keras_graph, ncnn_graph, NcnnParamDispatcher())
  File "/home/hx/.local/lib/python3.6/site-packages/keras2ncnn/keras_converter.py", line 850, in parse_keras_graph
    ncnn_helper)
  File "/home/hx/.local/lib/python3.6/site-packages/keras2ncnn/keras_converter.py", line 422, in ReLU_helper
    if layer['layer']['config']['threshold'] != 0:
KeyError: 'threshold'

```

**Comment by MarsTechHAN** [comment]
Fixed in 1960b9c, but the debugger crashed during the inference, so I am not 100% sure that it is dixed.

**Comment by MarsTechHAN** [comment]
![image](https://user-images.githubusercontent.com/10462805/102447148-629b7b80-406a-11eb-8c96-7b7fdb6fb8a1.png)
Fixed in ec09e66.
